src/main/java/com/google/javascript/clutz/DeclarationGenerator.java

```diff
@@ -989,7 +995,7 @@ private void declareNamespace(
 
       ObjectType oType = symbol.getType().toMaybeObjectType();
 
-      if (oType != null) {
+      if (oType != null && !isGoogNamespace) {
```

Comment:
add a comment here, that we skip this pass for goog namespace because of global oddity. I think we should come back to this one day.